extensions/wikia/Follow/FollowHelper.class.php

```diff
@@ -243,14 +253,30 @@ static public function watchBlogListing( &$article, &$user, $text, $summary, $mi
 				$title = Title::makeTitle( NS_BLOG_LISTING, $title );
 				$related[] = ucfirst( $title->getDBKey() );
 			}
+
 			self::emailNotification(
-				$article->getTitle(),
+				$postTitle,
 				$related,
 				NS_BLOG_LISTING,
 				$user,
 				self::LOG_ACTION_BLOG_POST,
 				wfMessage( 'follow-bloglisting-summary' )->text()
 			);
+
+			$userBlogTitleText = $postTitle->getBaseText();
+			$userBlogTitle = Title::makeTitle( NS_BLOG_ARTICLE, $userBlogTitleText );
+			if ( $userBlogTitle && $userBlogTitle->exists() ) {
```

Comment:
I don't think there's a case where `$userBlogTitle` would be null here, `Title::makeTitle` looks like it always returns a Title object. Can we just call `$userBlogTitle->exists()` here?